### PR TITLE
Reformat documentation for AppLoaderPlugin.ts and Loader.ts

### PR DESCRIPTION
**Pull request by omer8374**
#### Description of change

#7159

#### Pre-Merge Checklist

- [x] Documentation is changed or added
- [X] Lint process passed (`npm run lint`)

### CODE REVIEW

File: packages/loaders/src/Loader.ts [hunk 1]
```diff
@@ -668,9 +631,8 @@ class Loader
     private static _shared: Loader;
     /**
      * If this loader cannot be destroyed.
-     * @default false
      */
-    private _protected: boolean;
+    private _protected = false;
```
Comment (ShukantPal) [inline packages/loaders/src/Loader.ts:635]: This is technically a code change.

Comment (omer8374) [reply packages/loaders/src/Loader.ts:635]: Should I change it back? Or is there something else I need to do?

Comment (bigtimebuddy) [reply packages/loaders/src/Loader.ts:635]: @omer8374 if you could change and it back and push, that would be great. Thanks. In general, we want these documentation fixes to not touch any code, even if it seems small and trivial.

Comment (omer8374) [reply packages/loaders/src/Loader.ts]: @bigtimebuddy thanks I changed it back.

**Review by ShukantPal — APPROVED** [review]

**Comment by ShukantPal** [comment]
Thanks!

**Review by Zyie — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Thank you @omer8374

**Pull request merged**